voc/python/modules.py

```diff
@@ -14,7 +14,8 @@
 
 class ModuleBlock(Block):
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
```

Comment:
At column 12: [(E124) closing bracket does not match visual indentation](http://.../E124)